CAM/InvalidateVehicleIdleCam.md

```diff
@@ -0,0 +1,23 @@
+---
+ns: CAM
+aliases: ["_0x9E4CFFF989258472"]
+---
+## INVALIDATE_VEHICLE_IDLE_CAM
+
+```c
+// 0x9E4CFFF989258472 0x65DDE8AF
+void INVALIDATE_VEHICLE_IDLE_CAM();
```

Comment:
Seems that this name doesn't fit joaat hash (i.e. that's not a real name of this native). In this case you should add a `_` prefix to this name.

Reply:
Disquise long time ago, yea I was unsure if it needed a _ or not.